CHANGELOG.md

```diff
@@ -6,7 +6,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [unreleased]
 
-## [0.5.1] - 2022-03-02
+### Breaking Change
+- Refactored emailverify recipe interfaces.
```

Comment:
Put a bit more info about what the refactor is about. With an example. Also link to the relevant github issue